src/main/scala/com/starrocks/connector/spark/read/StarRocksRead.scala

```diff
@@ -0,0 +1,173 @@
+// Modifications Copyright 2021 StarRocks Limited.
```

Comment:
name the `StarRocksRead` to `StarRocksScan`?

Reply:
rename as StarRocksBatchScan